src/CodeGeneration/ApiGenerator/Domain/ApiQueryParameters.cs

```diff
@@ -104,6 +104,7 @@ public string TypeLowLevel
 				{
 					case "boolean": return "bool?";
 					case "list": return "string[]";
+					case "int":
```

Comment:
I'm wondering if we ought to raise an issue for this to change from `int` to `integer`?

Reply:
issue or pull request 👍

Reply:
Looking at the specs, it looks like they no longer contain `"integer"`, so I'll remove that case